client/lcd/lcd_test.go

```diff
@@ -435,10 +435,10 @@ func TestValidatorsQuery(t *testing.T) {
 	// make sure all the validators were found (order unknown because sorted by operator addr)
 	foundVal := false
 	pkBech := sdk.MustBech32ifyConsPub(pks[0])
-	if validators[0].PubKey == pkBech {
+	if validators[0].ConsPubKey == pkBech {
 		foundVal = true
 	}
-	require.True(t, foundVal, "pkBech %v, operator %v", pkBech, validators[0].Operator)
+	require.True(t, foundVal, "pkBech %v, operator %v", pkBech, validators[0].ConsPubKey)
```

Comment:
Why does this change the `printf`? The error is wrong now

Reply:
Bad copy-pasta job -- good catch.